eng/tools/TypeSpecValidation/test/tsv-test-host.ts

```diff
@@ -0,0 +1,11 @@
+import { TsvHost } from "../src/tsv-host.js";
+
+export const TsvTestHost: TsvHost = {
+    gitOperation: (folder: string) => {
+        return {
+            status: () => {return {modified:[`${folder}\n`], isClean: ()=>{return false}}},
```

Comment:
We will need some way to configure the behavior of these mocks for each test.  We could use constructor arguments or APIs after the mock is created.